index.d.ts

```diff
@@ -36,6 +36,14 @@ declare module 'i18next' {
   }
 }
 
+declare global {
+  namespace JSX {
+    interface IntrinsicAttributes {
+      i18nIsDynamicList?: boolean;
```

Comment:
Why is this TS change necessary?

Reply:
It allows you to pass the i18nIsDynamicList to the React.Fragment component.